packages/markdown-docx/src/CiceroMarkToOOXML/rules.js

```diff
@@ -78,4 +78,39 @@ const HEADING_RULE = (value, level) => {
     `;
 };
 
-module.exports = { TEXT_RULE, EMPHASIS_RULE, HEADING_RULE };
+/**
+ * Inserts a variable.
+ *
+ * @param {string} title Title of the variable. Eg. receiver-1, shipper-1
+ * @param {string} tag   Name of the variable. Eg. receiver, shipper
+ * @param {string} value Value of the variable
+ * @param {string} type  Type of the variable - Long, Double, etc.
+ * @returns {string} OOXML string for the variable
+ */
+const VARIABLE_RULE = (title, tag, value, type) => {
+    return `
+      <w:sdt>
+        <w:sdtPr>
+          <w:rPr>
+            <w:color w:val="000000"/>
+            <w:sz w:val="24"/>
+            <w:highlight w:val="green"/>
```

Comment:
Remove this for now. We will discuss later if we want to highlight variables or now.